src/routes/openapi/rapidoc/+page.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+    import { onMount } from "svelte";
+
+    onMount(async () => {
+        // @ts-ignore
```

Comment:
![Screenshot from 2023-09-16 22-53-07](https://github.com/tokitou-san/CoreProject-Frontend/assets/114811070/0a23c36a-cd70-4908-b18b-dd2cbb97e7bc)

https://github.com/rapi-doc/RapiDoc/pull/862
cause of this I'm not gonna shoo you XD
but they aren't even reviewing, that makes me feel like to shoo you